### PR TITLE
Roxygen

### PR DESCRIPTION
**Pull request by olivroy**
Closes #778. Part of #776

(the last commit didn't want to show up on the PR)

Imo, this simplifies the documentation by using markdown. Adding () to signify a function, to make it distinct from arguments.
roxygen2md::roxygen2md() + tweaks
Remove broken links
Simplify documentation for data https://r-pkgs.org/data.html#sec-documenting-data

A round of doc improvement could probably be done prior to release.

as an example, see how on the [tmap website](https://r-tmap.github.io/tmap/reference/tm_text.html#arguments), the group argument doc is not rendered correctly, without Roxygen(markdown = TRUE)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,13 +68,10 @@ install_github("r-tmap/tmap")
 pak::pak("r-tmap/tmap")
 
 # Or from r-universe
-options(repos = c(
-  tmap = 'https://r-tmao.r-universe.dev',
-  CRAN = 'https://cloud.r-project.org'))
-install.packages('tmap')
+install.packages('tmap', repos = c('https://r-tmap.r-universe.dev', 'https://cloud.r-project.org'))
```
Comment (olivroy) [inline README.md:71]: @janmarvin fixed here, oops!

**Comment by mtennekes** [comment]
Excellent, thx! There is one small merging conflict. Which one shall we use @olivroy ?

**Comment by mtennekes** [comment]
Oh, and when you are still working on DESCRIPTION @olivroy , please don't forget to put your name on the @Authors@R list :-) (fully deserved)

**Comment by olivroy** [comment]
@mtennekes Merged conflict + added myself as ctb while I was at it.

Thanks!

**Pull request merged**